examples/Cassie/run_dircon_squatting.cc

```diff
@@ -633,6 +633,8 @@ void DoMain(double duration, int max_iter, string data_directory,
     }
   }
 
+  trajopt->CreateVisualizationCallback("examples/Cassie/urdf/cassie_v2.urdf");
```

Comment:
`cassie_v2.urdf` should be replaced by `cassie_fixed_springs.urdf`

Reply:
Additionally, we can add `CreateVisualizationCallback` to the `run_dircon_walking` example (line 908) as well

Reply:
Done.